subiquity/controllers/ssh.py

```diff
@@ -33,12 +33,21 @@ def __init__(self, message, output):
 
 class SSHController(SubiquityController):
 
-    model_name = "ssh"
+    autoinstall_key = model_name = "ssh"
 
     def __init__(self, app):
         super().__init__(app)
         self._fetch_task = None
 
+    def load_autoinstall_data(self, data):
+        if data is None:
+            return
+        self.model.install_server = data.get('install_server', False)
+        self.model.authorized_keys = self.autoinstall_data.get(
+            'authorized-keys', [])
+        self.model.pwauth = self.autoinstall_data.get(
+            'allow-pw', not self.model.authorized_keys)
```

Comment:
Policy decision?  Is this documented that one cannot set password and import ssh keys?

Reply:
Uh, the idea is only that the default of allow-pw is False iff a key is supplied. And I think that's what this code does but I've been thinking about udev too much to be sure.